ext/Common.jl

```diff
@@ -68,6 +71,67 @@ function preprocess_aerosol_data(X::DataFrame)
     return X
 end
 
+function get_ARG_act_frac(data_row::NamedTuple)
+
+    FT = Float32
```

Comment:
Would it be possible to pass in the FT and the parameter structs as arguments, instead of defining them here?